automatic_data_generation/training/trainer.py

```diff
@@ -214,10 +231,14 @@ def compute_loss(self, logp, bow, target, length, mean, logv, logc, y,
 
         # labels loss
         if self.model.conditional == 'supervised':
+            if 'None' in self.dataset.i2int:
+                none_idx = self.dataset.int2i['None']
+            else:
+                none_idx = -1
```

Comment:
for better readability and avoid confusion with index "last element of list" in Python, I'd put -100 (which is the default value of the `ignore_index` argument of `torch.nn.NLLLoss`)